ecommerce/api.py

```diff
@@ -135,6 +168,10 @@ def apply_user_discounts(request):
             discount = user_discount.discount
 
     if discount:
+        # check for product specificity in the discount
```

Comment:
Could you add a Returns section in the docstring for this function

Reply:
Done in https://github.com/mitodl/mitxonline/pull/708/commits/fe65eb05b4c6f846d4b157a46804f0f800656674